test/fixtures/VatRegistrationFixture.scala

```diff
@@ -24,7 +24,8 @@ import uk.gov.hmrc.play.http._
 
 trait VatRegistrationFixture {
 
-  val IM_A_TEAPOT: Int = 418
+  val OK = 200
```

Comment:
play.api.http.Status trait defines all HTTP status codes as vals, we could import play.api.http.Status._ and have codes available